webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -42,34 +41,22 @@ const Layout = ({
       onExpand={expandLayoutMenus}
     >
       <VerticalNav.Masthead>
-        <VerticalNav.Brand
-          title={data.brand}
-          iconImg={data.logo}
-          href={data.root}
-        />
-        <TaxonomySwitcher
-          taxonomiesBool={data.taxonomies}
-          currentLocation={currentLocation}
-          locations={
-            data.taxonomies.locations ? data.locations.available_locations : []
-          }
-          onLocationClick={changeLocation}
-          currentOrganization={currentOrganization}
-          organizations={
-            data.taxonomies.organizations
-              ? data.orgs.available_organizations
-              : []
-          }
-          onOrgClick={changeOrganization}
-          isLoading={isLoading}
-        />
-        <UserDropdowns
-          notificationUrl={data.notification_url}
-          user={data.user}
-          changeActiveMenu={changeActiveMenu}
-          stopImpersonationUrl={data.stop_impersonation_url}
-          instanceTitle={data.instance_title}
-        />
+        <header className="pf-c-page__header pf-c-page" id="navbar-header">
+          <VerticalNav.Brand
+            title={data.brand}
+            iconImg={data.logo}
+            href={data.root}
+          />
+          <HeaderToolbar
+            data={data}
```

Comment:
how about:
```suggestion
      {...data}
```

to make props simpler in `HeaderToolbar`